autoload/neomake.vim

```diff
@@ -18,7 +18,9 @@ let s:need_errors_cleaning = {
     \ }
 let s:maker_defaults = {
             \ 'buffer_output': 1,
-            \ 'remove_invalid_entries': 0}
+            \ 'remove_invalid_entries': get(g:,'neomake_remove_invalid_entries', 0),
+            \ 'place_signs': get(g:, 'neomake_place_signs', 1),
+            \ 'open_list': get(g:, 'neomake_open_list', 0)}
```

Comment:
This would not pick up changed settings (unless you resource the file).
I think it should be moved to https://github.com/neomake/neomake/blob/3c52f9f27a9ecbf32ed5f79b5faf270ce4cf3ab8/autoload/neomake.vim#L439-L443.